assets/src/vo/date-time/datetime.js

```diff
@@ -294,7 +294,7 @@ export default class DateTime {
 	 * Given an indefinite number of DateTimes as arguments, this will return a
 	 * new DateTime that represents the latest point in time.
 	 * @param {...DateTime} datetimes
-	 * @return {DateTime} A new DateTime representing the latest point of time.
+	 * @return {DateTime|ServerDateTime} A new DateTime representing the latest point of time.
```

Comment:
Changes in the VO file should be made in a pull off master otherwise you increase the risk of conflicts for work in other branches off master.

Reply:
oh dang it... right...